scripts/uberenv/spack_configs/toss_3_x86_64_ib/packages.yaml

```diff
@@ -1,22 +1,16 @@
 packages:
   all:
+    # This defaults us to machine specific flags of ivybridge which allows
```

Comment:
Did you check if this works on rzalastor? I assume yes.

Reply:
I ran astyle --version and it didnt core dump, so thats a plus.